frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -349,13 +348,18 @@ export const PointClusterer: React.FC<PointClustererProps> = ({
             )}
             onclick={() => {
               //sets this pin as currently selected
+              const convertedProperty = convertToProperty(
+                m.properties,
+                m.position.lat,
+                m.position.lng,
+              );
               if (
                 m.properties.propertyTypeId === PropertyTypes.PARCEL ||
                 m.properties.propertyTypeId === PropertyTypes.SUBDIVISION
               ) {
-                dispatch(parcelsActions.storeParcelDetail(m.properties as IParcel));
+                dispatch(parcelsActions.storeParcelDetail(convertedProperty as IParcel));
```

Comment:
This was causing the error as when it tried to reload the selected property marker there was no lat/long present